src/stats.rs

```diff
@@ -0,0 +1,53 @@
+use anyhow::Result;
+use itertools::Itertools;
+use osmpbfreader::OsmId;
+use osmpbfreader::OsmObj;
+use std::collections::BTreeMap;
+use std::collections::HashMap;
+use std::io;
+use std::vec::IntoIter;
+
+use crate::util::filter_all_relations;
+
+fn sort_count(map: HashMap<&str, usize>) -> IntoIter<(&str, usize)> {
+    map.into_iter()
+        .sorted_by(|a, b| Ord::cmp(&b.1, &a.1))
+        .collect::<Vec<_>>()
+        .into_iter()
```

Comment:
`sorted_by` gibt schon einen `Iterator` zurück und alles was `collect().into_iter()` macht, ist einen container anlegen und sofort wieder wegwerfen.

```suggestion
    map.into_iter()
        .sorted_by(|a, b| Ord::cmp(&b.1, &a.1))
```